src/alg_utils.jl

```diff
@@ -195,3 +194,16 @@ end
 
 issplit(::StochasticDiffEqAlgorithm) = false
 issplit(::SplitSDEAlgorithms) = true
+
+function OrdinaryDiffEq.unwrap_alg(integrator::SDEIntegrator, is_stiff)
+  alg = integrator.alg
+  iscomp = typeof(alg) <: StochasticCompositeAlgorithm
+  if !iscomp
+    return alg
+  elseif typeof(alg.choice_function) <: AutoSwitch
+    num = is_stiff ? 2 : 1
+    return alg.algs[num]
+  else
+    return alg.algs[integrator.cache.current]
+  end
+end
```

Comment:
Just noticed that `unwrap_alg` is actually implemented as suggested in the comment above. So maybe just
```suggestion
function OrdinaryDiffEq.unwrap_alg(integrator::SDEIntegrator, is_stiff)
  return unwrap_alg(integerator, is_stiff)
end
```
? Or can `StochasticDiffEq.unwrap_alg` be removed?

Reply:
it's a typing issue and I think it's due to how DelayDiffEq is calling into it as well. So at least for now we need it, and we can continue to clean this up.